packages/client/hmi-client/src/components/workflow/ops/compare-datasets/tera-compare-datasets-drilldown.vue

```diff
@@ -262,60 +293,99 @@ function findDuplicates(strings: string[]): string[] {
 }
 
 async function createCharts() {
+	// FIXME: Temporary check, useChartSettings updates the state directly but the knobs are not updated
+	// If this isn't synced the chartSettings will be reverted to the previous state
+	if (!isEqual(props.node.state.chartSettings, knobs.value.chartSettings)) {
+		knobs.value.chartSettings = props.node.state.chartSettings;
+	}
+
```

Comment:
The `useChartSetttings` is set up to `update-state` directly but not with the `knobs`. So this is a workaround so the knob watcher doesn't remove the update from the `update-state`. 

I am starting to think that for the operators in general we should just run `update-state` when the `knobs` watcher is triggered. We are using both ways in most operators. I think I recall there was some infinite loop cases when watching for certain knobs...(idk). But I think it'll be worth cleaning this up if we can